### PR TITLE
fix pipeline acceptance tests

### PR DESCRIPTION
**Pull request by oldgiova**
Ticket: QA-620
Requires: https://github.com/mendersoftware/mender-test-containers/pull/415

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -131,15 +131,14 @@ publish:image-multiplatform:mender:worker:
 
 publish:acceptance:
   stage: publish
-  except:
-    - /^saas-[a-zA-Z0-9.]+$/
-  image: golang:1.14-alpine3.11
+  rules:
+    - if: $CI_COMMIT_BRANCH =~ /^saas-[a-zA-Z0-9.]+$/
+      when: never
+    - when: on_success
```
Comment (lluiscampos) [inline .gitlab-ci.yml:137]: It is the same as:

```suggestion
  rules:
    - if: $CI_COMMIT_BRANCH !~ /^saas-[a-zA-Z0-9.]+$/
```

`when: on_success` is the default, but if you like it explicit you ca use:
```suggestion
  rules:
    - if: $CI_COMMIT_BRANCH !~ /^saas-[a-zA-Z0-9.]+$/
      when: on_success
```

**Review by lluiscampos — APPROVED** [review]
Nice!

I have a suggestion to tidy the `rules` keyword once we are modernizing it from `except`.

File: .gitlab-ci.yml (hunk 1 above)
Comment (oldgiova) [reply .gitlab-ci.yml]: Right! Thanks for the hints!

**Comment by oldgiova** [comment]
@mender-test-bot start pipeline

**Pull request merged**